docs/platform/concepts/vpcs.md

```diff
@@ -0,0 +1,82 @@
+
+A VPC is a logically isolated section of a cloud provider's network, which makes it a
+private network within a public cloud. It's a secure customizable network environment that
+you define and control to deploy and manage resources.
+
+### VPC characteristics
+
+- Isolation: Each VPC operates independently from other VPCs, ensuring secure separation.
+- Security: Control over network traffic and isolation.
+- Internet connectivity: Control whether the VPC connects to the internet via internet
+  gateways or remains isolated.
+- Network control: Configure route tables, network gateways, and security settings.
+- Customizable IP range: You can define your own IP address range (CIDR block).
+- Subnets: Divide the VPC into smaller sub-networks (subnets) for organizing resources
+  based on availability zones or functional groups.
+- Flexibility: Custom network architecture tailored to your application's needs.
+- Scalability: Expand or modify the network as demand grows.
+
+### VPC components
+
+- Subnets: Represent smaller public or private networks within the VPC.
+- [Peering connection](/docs/platform/howto/list-vpc-peering): Connect VPCs for
+  intercommunication.
+- NAT (Network Address Translation) gateway: Allows outbound internet access for private
+  subnets.
+- Internet gateway (IGW): Enables public traffic to access the internet.
+- Security groups: Represent firewall rules controlling inbound and outbound traffic for
+  resources.
+- Route tables: Specify how traffic is directed within the VPC.
+- Network Access Control Lists (NACLs): Constitute an extra layer of security at the subnet
+  level
+
+### VPC use cases
+
+- Data isolation: Keeping sensitive data within a private network
+- Hosting applications: Deploying scalable web and database applications
+- Multi-tier architecture: Separating application layers (web, app, database) within
+  distinct subnets
+- Hybrid cloud architecture: Connecting on-premises networks to the cloud securely
+
+## VPC types
+
+The Aiven Platform allows creating and using two types of VPCs, which differ in scope:
+project-wide VPCs and organization-wide VPCs.
+
+### Project VPCs
+
+A project VPC is a VPC that spans a single Aiven project within your Aiven organization.
+A project-wide VPC allows all resources in that project to interconnect and share a common
+VPC network, simplifying network management and promoting consistency across your Aiven
+project's services.
+
+Learn how to
+[create and manage projects VPCs in Aiven](/docs/platform/howto/manage-project-vpc).
+
+### Organization VPCs <LimitedBadge/>
+
+An organization VPC is a VPC that spans multiple Aiven projects within your Aiven
+organization. An organization-wide VPC allows different projects to share a centralized
+network infrastructure while maintaining isolation and control.
+
+Learn how to
+[create and manage organization VPCs in Aiven](/docs/platform/howto/manage-organization-vpc).
+
+## Learn more
```

Comment:
Change to <RelatedPages/> component? ref https://github.com/aiven/aiven-docs/pull/681